azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -271,10 +273,15 @@ public void setGlobalProps(Props globalProps) {
     // Every 2 mins clean the recently finished list
     private static final long RECENTLY_FINISHED_INTERVAL_MS = 2 * 60 * 1000;
 
+    // Every 5 mins kill flows running longer than allowed max running time
+    private static final long LONG_RUNNING_FLOW_KILLING_INTERVAL_MS = 5 * 60 * 1000;
```

Comment:
I don't have a strong opinion either way. Since the name and the comment clearly indicate the unit is in minutes, I think it is a fine choice.